test/core/TestingFunctionMacro_UnitTests.cmake

```diff
@@ -978,6 +995,72 @@ FUNCTION(UNITTEST_TRIBITS_ADD_TEST_BASIC)
   UNITTEST_NOT_HAS_SUBSTR_CONST(TRIBITS_SET_TEST_PROPERTIES_INPUT
     "DISABLED")
 
+  MESSAGE("<fullTestName>_SET_RUN_SERIAL=ON")
+  SET(TRIBITS_SET_TEST_PROPERTIES_CAPTURE_INPUT ON)
+  SET(${PACKEXEN}_SET_RUN_SERIAL ON)
+  TRIBITS_ADD_TEST( ${EXEN}
+    ADDED_TESTS_NAMES_OUT ${EXEN}_TEST_NAMES )
+  UNITTEST_COMPARE_CONST( ${EXEN}_TEST_NAMES
+    "PackageA_SomeExec" )
+  UNITTEST_COMPARE_CONST(
+    TRIBITS_ADD_TEST_ADD_TEST_INPUT
+    "${PACKEXEN};${PACKEXEN_PATH}"
+    )
+  UNITTEST_COMPARE_CONST( MESSAGE_WRAPPER_INPUT
+    "-- PackageA_SomeExec: Added test (BASIC, PROCESSORS=1, RUN_SERIAL)!" )
+  UNITTEST_HAS_SUBSTR_CONST(TRIBITS_SET_TEST_PROPERTIES_INPUT
+    "RUN_SERIAL")
```

Comment:
`RUN_SERIAL;ON`?

Reply:
No, this should look for `RUN_SERIAL;ON` in this case just to be sure.  I will create a new TriBITS PR that looks for `RUN_SERIAL;ON` in this check.

Reply:
Addressed in TriBITSPub/TriBITS#330